contracts/plugins/assets/convex/CvxStableMetapoolCollateral.sol

```diff
@@ -91,13 +82,41 @@ contract CvxStableMetapoolCollateral is CvxStableCollateral {
         return (low, high, 0);
     }
 
+    /// Can revert, used by `_anyDepeggedOutsidePool()`
+    /// Should not return FIX_MAX for low
+    /// Should only return FIX_MAX for high if low is 0
+    /// @return low {UoA/tok} The low price estimate
+    /// @return high {UoA/tok} The high price estimate
+    function tryPairedPrice() public view virtual returns (uint192 low, uint192 high) {
+        uint192 p = chainlinkFeed.price(oracleTimeout); // {UoA/tok}
```

Comment:
not sure what the units should be here but i don't think they're `{UoA/tok}`. convex pool units seem to be even more abstract than our already 3-level-deep abstractions...

Reply:
I think `{UoA/tok}` is correct especially since this is the stable collateral. (And I agree about the Convex/Curve units, damn)

Reply:
Good point, these should be `{UoA/pairedTok}`